### PR TITLE
add Readme in Chinese.

### PR DESCRIPTION
**Pull request by gabrielchen0**
Translated Readme into Simplified Chinese and Traditional Chinese.

### CODE REVIEW

File: README.zh.sim.md [hunk 1]
```diff
@@ -0,0 +1,263 @@
+## npm-check  [![Build Status](http://img.shields.io/travis/dylang/npm-check.svg)](https://travis-ci.org/dylang/npm-check) [![npm-check](http://img.shields.io/npm/dm/npm-check.svg)](https://www.npmjs.org/package/npm-check)
+
+> 检查需要更新的、不正确的、以及无用的 npm 依赖模块。
+
+[English](/README.md) | [简体中文](/README.zh.sim.md) | [繁體中文](/README.zh.tra.md)
+
+<img width="796" alt="npm-check -u" src="https://cloud.githubusercontent.com/assets/51505/9569917/96947fea-4f48-11e5-9783-2d78077256f2.png">
+
+### 优秀之处
```
Comment (lijunle) [inline README.zh.sim.md]: 请确保每句话都以句号结束，或者都*不*以句号结束。以保持统一。

File: README.zh.sim.md [hunk 2]
```diff
@@ -0,0 +1,263 @@
...
+  semverValid: '4.6.1',                 // 已安装的版本是否符合语义化版本规则（SemVer）？
+  easyUpgrade: true,                    // 在仅使用 `npm install` 命令的情况下，是否能够对这个模块进行更新？
+  bump: 'minor',                        // 在得到最新版本时，需要哪种 bump？ 比如 patch, minor, major。
+  unused: false                         // 这个模块是否不再使用了？
+},
+```
+
+在命令行中添加 `--debug` 参数，你也能看到这些。
+
+### 制作 `nom-check` 的原因
```
Comment (lijunle) [inline README.zh.sim.md]: npm-check

**Comment by lijunle** [comment]
AFAIK, the English readme is generated from [some templates](https://github.com/dylang/npm-check/tree/master/templates/readme).

@dylang Are you going to revert the generate process back to raw readme markdown file? I remember you mentioned this idea somewhere.

File: README.zh.sim.md (hunk 2 above)
Comment (itisbenjamin) [inline README.zh.sim.md]: 對，這個我也想了很久，不知道還有沒有其他人有更好的建議。

File: README.zh.sim.md (hunk 1 above)
Comment (itisbenjamin) [inline README.zh.sim.md]: 這點太不細心了，感謝指出，已經糾正。

**Comment by amilajack** [comment]
@gabrielchen0 i can fix this merge conflict if you give me permission to write to this. There should be some checkbox on this page that you need to check

**Comment by amilajack** [comment]
Closing due to no response

**Pull request closed without merging**